src/addRes.js

```diff
@@ -1,11 +1,18 @@
-export default function styleResField() {
-  const resField = document.querySelectorAll('.cont-res-dis li');
-
-  resField.forEach((el, i) => {
+export default function populate(data) {
+  const list = document.querySelector('.cont-res-dis');
+  list.innerHTML = null;
+  data.forEach((el, i) => {
+    const userScore = document.createElement('li');
+    userScore.className = 'user-scr';
```

Comment:
- [ ] As stated in the Javascript best practices guide please change all regular functions to es6 arrow function syntax.

Reply:
This issue is well resolved! ✔️